test/integration/generated/additionalProperties/package.json

```diff
@@ -79,7 +81,7 @@
     "unit-test:node": "cross-env TEST_MODE=playback npm run integration-test:node",
     "unit-test:browser": "echo skipped",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "integration-test:node": "mocha -r esm --require ts-node/register --timeout 1200000 --full-trace test/*.ts --reporter ../../../common/tools/mocha-multi-reporter.js",
+    "integration-test:node": "dev-tool run test:node-ts-input -- --timeout 1200000 'test/*.ts'",
```

Comment:
We need to make sure that this is only generated like this when azure-sdk-for-js flag is true. Otherwise dev-tool is not going to be available and `dev-tool` can't be published